src/style.css

```diff
@@ -1 +1,100 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: bold;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You have a good set of fallback fonts here, but it's a good practice to also include a generic font family at the end, such as 'sans-serif' to ensure maximum compatibility.